.release-notes/next-release.md

```diff
@@ -12,7 +12,3 @@ Changes in Big Sur and xcode as of 12.5 have lead to an out of memory error with
 
 VM_FLAGS_SUPERPAGE_SIZE_ANY isn't required on earlier versions. It does however, improve performance when allocating. Usage of VM_FLAGS_SUPERPAGE_SIZE_ANY has been removed as it also doesn't work on newer M1 based machines and thus, is on its way out in general.
 
-## Change minimum supported MacOS API version to 10.14
-
-Our policy is to support the last 3 MacOS API versions with ponyc. We are now only supporting MacOS API versions 10.14 and up. Attempting to use ponyc on an earlier version of MacOS is not guaranteed to work and is no longer supported.
-
```

Comment:
This as yet unreleased change is superseded by our new change.

Reply:
I think we should still spell this policy out in the changelog. Supporting the same set of versions that Apple does makes sense, and we haven't tested these last changes on High Sierra and below.

Reply:
We haven't tested on anything other than Big Sur really. That's our CI. Our previous "3 versions" was that min API version which... appears to be moot so there's no need for the CHANGELOG entry as far as I can see.

Reply:
Okay, I see. This is probably more in the sense of someone coming with a bug on High Sierra and being able to say "Well, we don't really have a way of getting a machine that runs anything older than Mojave" plus "Mojave is the latest macOS version we support, sorry"